comp/core/flare/flare.go

```diff
@@ -113,7 +114,7 @@ func (f *flare) onAgentTaskEvent(taskType rcclienttypes.TaskType, task rcclientt
 		return true, fmt.Errorf("User handle was not provided in the flare agent task")
 	}
 
-	filePath, err := f.Create(nil, nil)
+	filePath, err := f.Create(nil, 0, nil)
```

Comment:
Question: Do we see RC using `provider-timeout` in the feature?

Reply:
Why not, probably not really needed for now but could be useful at some point